pom.xml

```diff
@@ -23,6 +23,14 @@
         </repository>
     </repositories>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.igniterealtime.openfire</groupId>
```

Comment:
I don't think you need to explicitly define a dependency on Openfire. That should be defined in the parent pom (`org.igniterealtime.openfire:plugins`)